static/app/actionCreators/tags.tsx

```diff
@@ -130,10 +128,6 @@ export function fetchTagValues({
     query.includeSessions = '1';
   }
 
-  if (includeReplays) {
```

Comment:
can you add this back in? that way it will only pull from the replays dataset

Reply:
yeah, idk why i removed this tbh. testing something? derp